### DISCUSSION TITLE
Seeking information about pockethost.io

### DISCUSSION THREAD

**Discussion by merajhossain028** [pocketbase/pocketbase#2344] [Q&A] [answered]
Can anyone please tell me the disadvantages of pockethost.io [[https://pockethost.io/](url)](https://pockethost.io/](url)?fbclid=IwAR0dFRQ-Pqy4PFVoqyN38jcjHOM18ulcwkVI5VLFJMH8pr2tbNAoJUeWMNo). Is it totally free and good for production? I have gone through the official documentation of pockethost.io but didn't get any clearance. 🙁

**Comment by ganigeorgiev** [comment, marked as answer]
You may want to check their official repo for issues and discussions in https://github.com/benallfree/pockethost/discussions.
